agent/whatweb_agent.py

```diff
@@ -230,7 +230,11 @@ def _prepare_ip_targets(self, message: msg.Message) -> List[IPTarget]:
         if mask is None:
             network = ipaddress.ip_network(f"{host}")
         else:
-            version = message.data.get("version")
+            try:
+                ip = ipaddress.ip_address(host)
```

Comment:
Nope, first get the version from message.data.get("version"), then if its None for the rest of your code

Reply:
Hay Hay Captain.